app/subscriber/src/features/home/Home.tsx

```diff
@@ -40,46 +48,12 @@ export const Home: React.FC = () => {
           selected: selectedContent,
         },
       }));
-      setLoading(false);
+      setIsLoading(false);
     },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
     [filter.startDate],
   );
 
-  const fetchResults = React.useCallback(
```

Comment:
Can access this logic through new hook, also splits the results appropriately for displaying previous 5 days